201-dsc-linux-azure-storage-on-ubuntu/metadata.json

```diff
@@ -1,7 +1,7 @@
 {
   "itemDisplayName": "DSCForLinux extension on a Ubuntu VM",
-  "description": "This template creates a Ubuntu VM and installs the DSCForLinux extension 2.0",
-  "summary": "This template creates a Ubuntu VM and installs the DSCForLinux extension 2.0",
-  "githubUsername": "lizzha",
-  "dateUpdated": "2016-03-15"
+  "description": "This template creates a Ubuntu VM and installs the DSCForLinux extension 2.70 or latest",
+  "summary": "This template creates a Ubuntu VM and installs the DSCForLinux extension 2.70 or latest",
+  "githubUsername": "balukambala",
```

Comment:
did you want to take ownership of these?

Reply:
yes, My manager take over the Linux extension for DSC. I am working on enhancements on DSC Linux extension. I am going to make few more changes to these templates.

Reply:
Cool - can you look at the build failures - it looks like the param file is not using generated values and the artifacts are not using BPs for the repo...

Reply:
Param file has fixed values and I see this error in the build output:

"error": "Error: Command failed: /bin/sh -c az group deployment create  --resource-group qstci-78841b45-b6ce-e804-fc02-14b048f55cf7 --template-file temp/435d7257-fdea-3600-79db-4cf530ccd4d4 --parameters temp/903447eb-50a6-1f1d-62a6-cfe380add187 -o json\nERROR: Deployment failed. {\r\n  \"error\": {\r\n    \"code\": \"StorageAccountAlreadyTaken\",\r\n    \"message\": \"The storage account named dscuniqueaccount1 is already taken.\"\r\n  }\r\n}\n"

It looks like storage account is already exist under this resource group. Are you expecting me to make these parameters dynamic?

Reply:
Yes, see the contribution guide for details...

Reply:
Build is successful. Please let me know if there is any pending here.